task.sql

```diff
@@ -38,3 +38,18 @@ CREATE TABLE ProductInventory (
     PRIMARY KEY (ID)
 );
 --rollback DROP TABLE ProductInventory;
+
+--changeset mate.academy:5 label:0.0.2
```

Comment:
The reason of the error is the label
```suggestion
--changeset mate.academy:5 labels:0.0.2
```